cob_hardware_config/cob4-2/config/base_controller.yaml

```diff
@@ -19,8 +19,8 @@ twist_controller:
       spring: 15.0 #10.0
       damp: 2.75 #2.5
       virt_mass: 0.1 #0.1
-      d_phi_max: 12.0 #100.0
-      dd_phi_max: 17.0 #100.0
+      d_phi_max: 10.0 #100.0
+      dd_phi_max: 40.0 #100.0
```

Comment:
These are the velocity (`d_phi_max`) and acceleration (`dd_phi_max`) limits of the steer controller.
(https://github.com/ipa320/cob_robots/blob/hydro_dev/cob_hardware_config/cob4-2/config/base/MotionCtrl.ini#L64)